cmd/skaffold/app/cmd/cmd.go

```diff
@@ -115,10 +115,6 @@ func NewSkaffoldCommand(out, err io.Writer) *cobra.Command {
 				}
 			default:
 			}
-
-			if shutdownAPIServer != nil {
```

Comment:
why did we have to move this to the RunE sections? Doesn't the `PersistentPostRun` good enough? I don't understand this change

Reply:
I explained this in the PR description. Maybe you missed it. Let me know if you are still confused.

Reply:
ooops, busted! I didn't read your description, I got it now, thank you :D

Reply:
One thing: a comment might be useful - so that people don't factor it out accidentally 6 months down the line this seeming duplication!